README.md

```diff
@@ -54,6 +54,28 @@ Magento 2
     - Open Setup Klaviyo User from the sidebar
     - Enter the username, password, and email for the credentials
     - Save config
+
+
+## Making updates
+1) Follow Klaviyo's standard process of making a pull request and getting it reviewed before merging.
+2) Update CHANGELOG.md. Details on formatting the changelog (including categorizing changes) can be found here: [keepachangelog.com](https://keepachangelog.com/en/1.0.0/)
+    1) If this is a change that will not immediately get sent along to Magento i.e. not a version update:
```

Comment:
what do you mean not get sent along to magento?  u mean packagist?

Reply:
Yea Magento marketplace and Packagist